code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -70,15 +70,15 @@
 		if(HAS_TRAIT(L, TRAIT_VENTCRAWLER_NUDE) || HAS_TRAIT(L, TRAIT_VENTCRAWLER_ALWAYS))
 			. += span_notice("Alt-click to crawl through it.")
 
-/obj/machinery/atmospherics/New(loc, process = TRUE, setdir, init_dir = ALL_CARDINALS)
+/obj/machinery/atmospherics/Initialize(mapload, process = TRUE, setdir, init_dir = ALL_CARDINALS)
```

Comment:
I feel like there was some reason for this, but I don't quite remember it. Fuck

Reply:
On second thought, it'd be insane to do this without doing all the atmos machines at once